ProfilerGUI/Source/Shared/ProfilerConstants.cs

```diff
@@ -0,0 +1,48 @@
+﻿namespace ProfilerGUI.Source.Shared
+{
+    /// <summary>
+    /// Constants used when running the profiler.
+    /// </summary>
+    public static class ProfilerConstants
```

Comment:
This class does not handle .NET Core variable names

Reply:
created #31 for that